Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "multipart"
 
-version = "0.10.2"
+version = "0.10.3"
```

Comment:
This probably should be `0.11.0`. As a convention, PRs generally seem to not bump the version number directly and instead leave it up to the package maintainer, but it doesn't matter much to me.